packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts

```diff
@@ -30,17 +30,8 @@ import {
 
 export abstract class BaseTemplateCommand extends SfdxCommandletExecutor<
   DirFileNameSelection
-  > {
-  private metadataType: MetadataInfo;
-
-  constructor(type: string) {
```

Comment:
we can keep the constructor (make type optional here) but your call. this is only for CLI templates and should retire gradually.